src/components/index.js

```diff
@@ -75,3 +75,4 @@ export { default as ColorUtils } from "../utils/colors-utils";
 export { default as Slider } from "./Slider/Slider";
 export { default as IconButton } from "./IconButton/IconButton";
 export { default as Flex } from "./Flex/Flex";
+export { default as MenuGridItem } from "./Menu/MenuGridItem/MenuGridItem";
```

Comment:
please add to publish components as well

Reply:
I'm not sure I understand the usage of each file correctly..
Also, should I add `useGridKeyboardNavigation` to the end of `published-components` as well?